### PR TITLE
Add localization doc to https://www.mapbox.com/android-docs/navigation

### PR DESCRIPTION
**Pull request by samfader**
Closes https://github.com/mapbox/android-docs/issues/759

I made a couple of small tweaks to the content (since it was taken from https://github.com/mapbox/mapbox-navigation-android/blob/master/localization.md), but the big issue that I can't seem to get past is the formatting at the bottom of the table:

image

Would love any and all thoughts!

### CODE REVIEW

File: src/pages/navigation/overview/index.md [hunk 1]
```diff
@@ -180,6 +180,55 @@ navigation?.locationEngine = locationEngine!!
...
+
+Turn instructions are primarily designed to be announced by either the Mapbox Voice API (powered by [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/SupportedLanguage.html)) or [TextToSpeech](https://developer.android.com/reference/android/speech/tts/TextToSpeech). By default, this SDK uses the Mapbox Voice API, which requires an Internet connection at various points along the route. If the Voice API lacks support for the turn instruction language or there is no Internet connection, TextToSpeech announces the instructions instead. 
+
+By default, distances are given in the predominant measurement system of the system region, which may not necessarily be the same region in which the user is traveling. To override the measurement system used in spoken instructions, set the `MapboxNavigationOptions.Builder#unitType` property when calculating the route with which to start navigation.
```
Comment (HeyStenson) [inline src/pages/navigation/overview/index.md]: > By default, distances are given in the predominant measurement system of the system region, which may not necessarily be the same region in which the user is traveling.

I think an example here could be instructive!

**Comment by HeyStenson** [comment]
> the big issue that I can't seem to get past is the formatting at the bottom of the table

@samfader Are you talking about how the content right after the table bumps up against the bottom of the table? If so, I think that this is down to a `dr-ui` issue, and may need to be addressed there. @colleenmcginnis @katydecorah does that sound right?

**Comment by colleenmcginnis** [comment]
We should address this with CSS. We could modify `site.css` in this repo, but i think it might make more sense to add a rule to https://github.com/mapbox/dr-ui/blob/master/src/css/docs-prose.css (which is applied to this repo in the [batfish config](https://github.com/mapbox/android-docs/blob/mb-pages/batfish.config.js#L81)) that will add some `margin-bottom` to tables. @katydecorah what do you think? Would you want to add that before your dr-ui release this week? 🚀

**Comment by katydecorah** [comment]
After a little 🕵️‍♀️, markdown tables in this repo are wrapped in a div. We have a rule in assembly to turn off the bottom margin on the last element to prevent awkward spacing at the end of a section. Since the table is wrapped in a div, it's technically the last element in that container and so the margin is getting overridden:

image

I push a commit to this PR to add `mb18` to the table wrapper - we should be good :)

File: src/pages/navigation/overview/index.md [hunk 2]
```diff
@@ -186,7 +186,34 @@ Turn instructions are announced in the user interface language when turn instruc
...
+<CodeLanguageToggle id="override-measurements" />
+<ToggleableCodeBlock
+
+java={`
+// Override measurement system in spoken instructions
+NavigationRoute.builder(context)
+    .accessToken(accessToken)
+    .origin(origin, bearing, tolerance)
+    .destination(destination)
+    .voiceUnits("imperial")
```
Comment (danesfeder) [inline src/pages/navigation/overview/index.md]: We can replace `"imperial"` with `DirectionsCriteria.IMPERIAL`, which is a constant we include in the SDK

File: src/pages/navigation/overview/index.md [hunk 3]
```diff
@@ -186,7 +186,34 @@ Turn instructions are announced in the user interface language when turn instruc
...
+    .build();
+`}
+
+kotlin={`
+// Override measurement system in spoken instructions
+NavigationRoute.builder(context)
+    .accessToken(accessToken)
+    .origin(origin, bearing, tolerance)
+    .destination(destination)
+    .voiceUnits("imperial")
```
Comment (danesfeder) [inline src/pages/navigation/overview/index.md]: Same here `DirectionsCriteria.IMPERIAL` 👍

**Comment by samfader** [comment]
@HeyStenson mind reviewing once more?

**Pull request merged**